k4FWCore/components/Writer.cpp

```diff
@@ -220,15 +225,16 @@ class Writer final : public Gaudi::Functional::Consumer<void(const EventContext&
     // deleted by the store (and later deleted by the Frame, triggering a double
     // delete)
     for (const auto& coll : frameCollections) {
+      debug() << "Taking ownership of collection " << coll << " from the IOSvc as it belongs to a Frame" << endmsg;
       DataObject* storeCollection;
       if (m_dataSvc->retrieveObject("/Event/" + coll, storeCollection).isFailure()) {
-        error() << "Failed to retrieve collection " << coll << endmsg;
-        return;
+        warning() << "Failed to retrieve collection " << coll << endmsg;
+        continue;
```

Comment:
```suggestion
        error() << "Failed to retrieve collection " << coll << endmsg;
        return;
```
Keep it as it is since it doesn't need to be changed and then maybe change in another PR?